core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -87,10 +87,35 @@ public static function getTemplate($strTemplate)
 	public static function getTemplateGroup($strPrefix)
 	{
 		$arrTemplates = array();
+		$arrOthers = array();
+
+		$arrMapper = array
+		(
+			'ce' => 'TL_CTE',
+			'form' => 'TL_FFL',
+			'mod' => 'FE_MOD',
+		);
 
 		// Get the default templates
 		foreach (TemplateLoader::getPrefixedFiles($strPrefix) as $strTemplate)
 		{
+			if ($strTemplate != $strPrefix)
+			{
+				list($k, $strKey) = explode('_', $strTemplate, 2);
+
+				if (isset($arrMapper[$k]))
+				{
+					foreach ($GLOBALS[$arrMapper[$k]] as $arrMappings)
+					{
+						if (isset($arrMappings[$strKey]))
+						{
+							$arrOthers[] = $strTemplate;
```

Comment:
can we call this `$arrConflicts`, `$arrMainTemplates` or something like that? "Other" does not say anything about what this contains.

Reply:
Changed in 7b6ce8525e7cc7b2119232a97173fad50f7e005f.